pkg/operation/botanist/component/hvpa/hvpa.go

```diff
@@ -145,6 +145,17 @@ func (h *hvpa) Deploy(ctx context.Context) error {
 					Resources: []string{"jobs"},
 					Verbs:     []string{"get", "list", "patch", "update", "watch"},
 				},
+				{
+					APIGroups: []string{"coordination.k8s.io"},
+					Resources: []string{"leases"},
+					Verbs:     []string{"create"},
+				},
+				{
+					APIGroups:     []string{"coordination.k8s.io"},
+					Resources:     []string{"leases"},
```

Comment:
I believe hvpa-controller needs to create leases and events only in the garden namespace. Hence, you could move these rules to a Role as the ClusterRole would allow thae actions in any namespace.

Reply:
done, thanks!